Index: squashfs/squashfs-tools/Makefile
===================================================================
--- squashfs.orig/squashfs-tools/Makefile
+++ squashfs/squashfs-tools/Makefile
@@ -2,7 +2,7 @@ INSTALL_DIR = /usr/local/bin
 
 INCLUDEDIR = .
 
-CFLAGS := -I$(INCLUDEDIR) -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_GNU_SOURCE -O2
+CFLAGS := $(RPM_OPT_FLAGS) -I$(INCLUDEDIR) -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_GNU_SOURCE -O2
 
 all: mksquashfs unsquashfs
 
